Doc/library/sys.rst

```diff
@@ -314,6 +314,35 @@ always available.
    yourself to control bytecode file generation.
 
 
+.. data:: _emscripten_info
+
+   A :term:`named tuple` holding information about the environment on the
```

Comment:
Why a named tuple? I don't see how indexing into the info is useful as it's all so disparate. Or is this for ease of coding in order to use `PyStructSequence`?

Reply:
Yes, that's correct. All info / flag collections in the sys module use ``PyStructSequence``, e.g. ``float_info``, ``thread_info``, and so on. The types are documented as named tuple.